pkgs/build-support/rust/build-rust-crate/configure-crate.nix

```diff
@@ -71,6 +71,8 @@ in ''
   export CARGO_PKG_VERSION_MAJOR=${builtins.elemAt version 0}
   export CARGO_PKG_VERSION_MINOR=${builtins.elemAt version 1}
   export CARGO_PKG_VERSION_PATCH=${builtins.elemAt version 2}
+  export NUM_JOBS=1
+  export RUSTC="rustc"
```

Comment:
I don't think it matters, as I believe this is meant to be `argv[0]` when `rustc` is called, and since we call `rustc` directly everywhere else, I believe this is what is expected in most build scripts.
Since we have `rustc` in the `builtInputs`, `rustc` is in the path, I think we're fine.

Reply:
I remember `rustc` to do a realpath on /proc/self/exe at some point, when trying to cheat with our rustup patch.